### DISCUSSION TITLE
Ubuntu 20 - App freezed on the first frame

### DISCUSSION THREAD

**Discussion by ronkah** [bordaigorl/rmview#95] [Q&A] [answered]
I installed rmview on my ubuntu 20.04 with the installation binary, and now I can connect to my remarkable1 but the app is stuck on the first frame and not updated to the current view of my device, anyone experienced it? Any idea what can I do?

**Comment by bordaigorl** [comment, marked as answer]
rmview does not currently support the remarkable software update 2.9 and above.
Support is coming, but cannot give a specific timeframe at the moment.
Watch this repo for updates.

**Reply by ronkah** [reply]
Oh... never crossed my mind that the update is the reason.. alright thanks! maybe Ill roll back the last update.

Thanks for all the hard work!
